google/cloud/functions/internal/wrap_request.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_INTERNAL_WRAP_REQUEST_H
+#define FUNCTIONS_FRAMEWORK_CPP_GOOGLE_CLOUD_FUNCTIONS_INTERNAL_WRAP_REQUEST_H
+
+#include "google/cloud/functions/http_request.h"
+#include "google/cloud/functions/version.h"
+#include <boost/beast/http.hpp>
+
+namespace google::cloud::functions_internal {
+inline namespace FUNCTIONS_FRAMEWORK_CPP_NS {
+
+using BeastRequest =
+    boost::beast::http::request<boost::beast::http::string_body>;
+
+/// Wrap a Boost.Beast request into a functions framework HTTP request.
+::google::cloud::functions::HttpRequest FromBeast(BeastRequest request);
```

Comment:
This could just be `MakeHttpRequest(BeastRequest)`. The from-type need not be part of the function name. Even if we wanted to overload this for other HTTP Request types, we could still do that.

No change required; just an observation. I realize this is internal.

Reply:
Done.